lib/steps.js

```diff
@@ -705,6 +705,13 @@ Then(/I get the(?: page's)?(?: JSON)? var(?:iable)?s and val(?:ue)?s/i, { timeou
   );
 });
 
+Then(/I expect the baseline PDF "([^"]+)" and the new PDF "([^"]+)" to be the same/i, async (existing_pdf_path, new_pdf_path) => {
+  return await wrapPromiseWithTimeout(
```

Comment:
Maybe a description here, take it or leave it

```suggestion
  /** Compare the text or fields of a baseline PDF and the newly downloaded PDF.
  * 
  * Variations:
  * Then I expect the baseline PDF "simple-doc-Baseline.pdf" and the new PDF "simple-doc.pdf" to be the same
  * Then I expect "simple-doc-Baseline.pdf" and "simple-doc.pdf" to be the same
  */
  
  return await wrapPromiseWithTimeout(
```

or

```suggestion
  /** Compare the text or fields of a baseline PDF and the newly downloaded PDF.
  * 
  * Variations:
  * Then I expect the baseline PDF "simple-doc-Baseline.pdf" and the new PDF "simple-doc.pdf" to be the same
  */
  
  return await wrapPromiseWithTimeout(
```